python/tskit/drawing.py

```diff
@@ -527,7 +527,7 @@ def draw_y_axis(
         self,
         upper=None,  # In plot coords
         lower=None,  # In plot coords
-        tick_positions=None,
+        ticks=(),  # A list of numerical positions or a dictionary of pos->label
```

Comment:
It's not obvious why this default value is treated differently to the others and this syntax will be opaque to some.

Reply:
Yeah, I had `None` originally, but then I need to swap None for e.g. `[]` and test it, which takes more code lines, so 🤷